### DISCUSSION TITLE
Where does Hyprland backup directory exist?

### DISCUSSION THREAD

**Discussion by Ramy-Ashraf** [prasanthrangan/hyprdots#1185] [Q&A] [answered]
I just want to know where the backups exist.

**Comment by dieBakterie** [comment]
If not specified it's `~/.configs/cfg_backups`

**Comment by Ramy-Ashraf** [comment]
Is it safe to clear it?

> If not specified it's `~/.configs/cfg_backups`

**Reply by kRHYME7** [reply]
```  Hyprdots backup clean ```

Tab to select

**Comment by dieBakterie** [comment, marked as answer]
If you don't need those yeah

**Reply by Ramy-Ashraf** [reply]
thanks alot
